azure/resource_groups/app/parameters/production.template.json

```diff
@@ -124,6 +124,9 @@
         "secretName": "DfeSignInApiEndpoint"
       }
     },
+    "ADMIN_ALLOWED_IPS": {
+      "value": "194.72.231.130/31,194.72.231.132/31,213.123.63.150,217.38.175.48/28,217.38.175.64/28,217.38.165.192/28,217.38.165.176/28"
```

Comment:
Would this be better in the keyvault? Would mean we can temporarily add our own IPs in case we need to do some debugging

Reply:
You'd still need to kick off another deployment. I'd prefer to keep it here, so it's clear what we deployed at a glance. I honestly think we have too much in the Key Vault. It makes us fragile to redeploys, and harder to reason about what settings we're running on for a given version.